utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -45,12 +47,15 @@ class HtmlParser private constructor(
       htmlContent = htmlContent.replace("&amp;quot;", "")
     }
 
-    val imageGetter = urlImageParserFactory.create(htmlContentTextView, entityType, entityId, imageCenterAlign)
+    val imageGetter = urlImageParserFactory.create(
+      htmlContentTextView, entityType, entityId, imageCenterAlign)
 
-    val htmlSpannable = HtmlCompat.fromHtml(htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()) as Spannable
+    val htmlSpannable = HtmlCompat.fromHtml(
+      htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()) as Spannable
 
     val spannableBuilder = SpannableStringBuilder(htmlSpannable)
-    val bulletSpans = spannableBuilder.getSpans(0, spannableBuilder.length, BulletSpan::class.java)
+    val bulletSpans = spannableBuilder.getSpans(
```

Comment:
Probably should break at ``=`` instead, ditto elsewhere.

Reply:
This is kind of hard to automate, I am not able to find a rule that can actually do this. We would probably have to create a custom rule, but given that ktlint is meant to follow the kotlin style guide, I am not sure if we should add custom rules for line breaks etc ?